Factory/PhpredisClientFactory.php

```diff
@@ -68,10 +68,14 @@ public function create($class, $dsn, $options, $alias)
 
         if (null !== $parsedDsn->getPassword()) {
             $client->auth($parsedDsn->getPassword());
+        } elseif (isset($options['parameters']['password'])) {
```

Comment:
Could you add unit tests for these cases please?

Reply:
I have added tests: https://github.com/snc/SncRedisBundle/pull/505/files#diff-faa94e9892a5c176d4921a4a202f29f5R677. Are not they enough? Please, let me know.

Reply:
Yes and that is great!
I think we also need to update SncRedisBundle/Factory/PhpredisClientFactoryTest.php.

Reply:
That's right. Updated.